src/CSSBattle/5_AcidRain/ZhangZhikai.html

```diff
@@ -0,0 +1,41 @@
+
+<div class="top">
+  <div class="shape">
+  </div>
+  <div class="shape bottom">
+  </div>
+  
+  </div>
+  
+  
```

Comment:
Consider removing unnecessary line breaks and indentations
```suggestion

<div class="top">
  <div class="shape">
  </div>
  <div class="shape bottom">
  </div>
  
  </div>
  
  
```

Reply:
50ed92b92dcb6d7fac9a9b6965067d8e1abde559